src/Weeder/Config.hs

```diff
@@ -3,14 +3,13 @@
 {-# language OverloadedStrings #-}
 {-# language RecordWildCards #-}
 
-module Weeder.Config ( Config(..), config ) where
+module Weeder.Config ( Config(..) ) where
 
 -- containers
 import Data.Set ( Set )
-import qualified Data.Set as Set
 
--- dhall
-import qualified Dhall
+-- toml
```

Comment:
```suggestion
-- toml-reader
```

I think? I should release the tool we use at CircuitHub to do this automatically!